include/tokenizer.h

```diff
@@ -8,15 +8,13 @@
 #include <variant>
 #include <vector>
 
-class Tokenizer
-{
+class Tokenizer {
     public:
         // Constructor/Destructor
         explicit Tokenizer(const std::string& source);
```

Comment:
I think it's important especially here to use `string_view`, the point of the tokenizer is literally to break down the view of a string